### ISSUE TITLE
We require setting scope for environment variables

### ISSUE THREAD

**Issue by bomoko** [Freedomtech-Hosting/ft-lagoon-php#2]
Currently it seems we can only set global variables. We should change this to include scope as well.

**Comment by bryangruneberg** [comment]
Agreed! The main project using this is Polydoc and for its implementation we only care about global project level variables, but I can see how this would be helpful for other places where ft-lagoon-php is being used! Will sort it out

**Comment by bryangruneberg** [comment]
@bomoko  - this sorts it out https://github.com/Freedomtech-Hosting/ft-lagoon-php/pull/6

I've already merged to main because I need it for things 😸
